src/Sulu/Bundle/SecurityBundle/Admin/SecurityAdmin.php

```diff
@@ -38,6 +38,14 @@ class SecurityAdmin extends Admin
 
     const EDIT_FORM_VIEW = 'sulu_security.role_edit_form';
 
+    /**
+     * Should be called after ContactAdmin.
+     */
+    public static function getPriority(): int
+    {
+        return -10;
```

Comment:
In other places we were using powers of 2 for similar values, because they easily allow to put other priorities in between them. And maybe we should make sure that there is enough room in between? So use something like `-1024`?